program/convert-string-to-pathcase/ConvertStringToPathcase.kt

```diff
@@ -0,0 +1,24 @@
+import java.util.Scanner
+
+fun main() {
+    print("Enter String : ")
+    println()
+    var min = readLine()
+
+    println("Camel Case: " + ConvertStringToPathcase(min!!.split(" ")))
+
```

Comment:
This will throw a NullPointerException while executing the code.
The reference variable should not be null

Reply:
Works fine

![Screenshot from 2023-01-02 22-54-23](https://user-images.githubusercontent.com/54644599/210262866-e24b50ad-5f72-45b9-9882-31db2812977d.png)